sdk/src/ldb/LDB.ls

```diff
@@ -53,47 +53,47 @@ package ldb {
 
                         trace("LDB Remote Debugger");
                         trace("Awaiting debug client connection");
-                        
+
                         // bind to local host and wait for client to connect
                         server = Server.listen("*", 8171);
 
                         trace("Debug client connected");
 
                     }
-                    else
+                    else if (msg != null)
                         trace(msg);
 
-                    
+
                 }
-                
+
             }
-            
+
             else {
-            
+
                 DebuggerClient.connect("127.0.0.1", 8171);
                 DebuggerClient.cmdSTEP();
-                
+
                 while(true) {
-                
+
                     DebuggerClient.update();
-                    
+
                 }
             }
-                
-        
+
+
             trace("Debugging session ended");
-            
+
         }
-        
+
     }
-    
+
     class InstanceTest {
-        
+
         public function doSomething() {
             trace("hi");
         }
-        
+
     }
-    
-    
```

Comment:
@themuflon can you revert the whitespace changes and put up a separate PR if the changes would make it more consistent with other files?